examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suppose. I want to keep it around to drive home the difference between `stretch` versus `row_span` and `col_span`